src/components/volume/volume.tsx

```diff
@@ -123,7 +123,7 @@ class Volume extends Component<any, any> {
    * @returns {boolean} - whether to render the component
    */
   _shouldRender(): boolean {
-    const isActive = !this.props.player.isImage();
+    const isActive = !(this.props.player.isImage() || this.props.player.isDoc());
```

Comment:
just a reminder - if we change the name to document in providers we should change it in other places as well to be aligned.
if we leave it as doc then its fine.